src/main/java/com/softserve/academy/spaced/repetition/domain/Account.java

```diff
@@ -27,18 +27,18 @@
     @Column(name = "account_id", nullable = false)
     private Long id;
 
-    @Column(name = "password")
+    @Column(name = "password", length = PASS_MAX_SIZE)
     @Size(message = PASS_SIZE_MESSAGE, min = PASS_MIN_SIZE, max = PASS_MAX_SIZE, groups = Request.class)
     private String password;
 
-    @Column(name = "email", unique = true, nullable = false)
+    @Column(name = "email", unique = true, nullable = false, length = EMAIL_MAX_SIZE)
```

Comment:
Is it necessary to set nullable = false, in annotation Column, if we have annotation NotNull below?